src/scss/components/card/vertical-card/_vertical-card-general.scss

```diff
@@ -10,13 +10,24 @@
         padding-top: 0;
       }
 
-      div:not(.su-card__button) {
-        // only target plain links, not button links
-        a.su-link {
-          @include modular-spacing('margin-top', 2);
-        }
+      // set for button
+      .su-card__button {
+        @include modular-spacing('margin-top', 2);
```

Comment:
We strive towards not using margin-top. I believe you can use a padding top here instead.

Reply:
I'm noting the difficulty of not having a wrapper around the super heading, h2, and paragraphs and the jira ticket I created to improve that: https://stanfordits.atlassian.net/browse/ADAPT-2057
So that we can have bottom padding instead of top padding. cc @jenbreese